### PR TITLE
BZ2066952 ComplianceCheckResult updates

### PR DESCRIPTION
**Pull request by sheriff-rh**
please cherry-pick to 4.6+

Per [BZ2066952](https://bugzilla.redhat.com/show_bug.cgi?id=2066952):
Preview: https://deploy-preview-43888--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-remediation.html

* Changed the title of `Managing Compliance Operator remediation` to `Managing Compliance Operator result and remediation`
* Changed the  name of `Inconsistent Remediations` to `Inconsistent ComplianceScan`
* Moved the `Filters for failed compliance check results` to the first section
* Included a table in `Filters for failed compliance check results` to describe each status
* Renamed `Filters for failed compliance check results` to `Filters for compliance check results`

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@Vincent056 please review: https://deploy-preview-43888--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-remediation.html

**Review by Vincent056 — COMMENTED** [review]
Thanks for adding the PR, I think it all looks very good, there is just one thing, I am wondering do you think it is better to use the title `Filters for failed compliance check results` or `Filters for compliance check results`? @sheriff-rh

**Comment by sheriff-rh** [comment]
That's a great point, @Vincent056 , I'll remove the word "failed" since we describe all possible outcomes in the table. Good catch, thanks!

**Review by Vincent056 — APPROVED** [review]
Thanks for updating the PR, it all looks good!

**Comment by pdhamdhe** [comment]
LGTM
@xiaojiey Hi, please help to review. Thanks

File: security/compliance_operator/compliance-operator-remediation.adoc [hunk 1]
```diff
@@ -24,8 +26,6 @@ include::modules/compliance-removing-kubeletconfig.adoc[leveloffset=+1]
 
 include::modules/compliance-inconsistent.adoc[leveloffset=+1]
 
-include::modules/compliance-filtering-failed-results.adoc[leveloffset=+1]
-
 [role="_additional-resources"]
```
Comment (mjpytlak) [inline security/compliance_operator/compliance-operator-remediation.adoc:29]: While you are here, please remove the the plain text from the link.

Per the guidance in [Jupiter Source files readiness Checklist](https://docs.engineering.redhat.com/display/JUP/Jupiter+Source+files+readiness+Checklist), these sections should contain links only. No plain text.

**Comment by mjpytlak** [comment]
Peer review LGTM.

**Pull request merged**